src/app/scripts/cac/control/cac-control-explore.js

```diff
@@ -327,6 +329,28 @@ CAC.Control.Explore = (function (_, $, Geocoder, MapTemplates, HomeTemplates, Ro
         }
     }
 
+    // Given desintations from the FindReachableDestinations app endpoint,
+    // display the returned list of places within the travelshed in the sidebar cards.
+    function listIsochronePlaces(destinations) {
+        showSpinner();
+        var $placeCards = $(options.selectors.placeCard);
+        // hide existing times to places now showing (if any)
+        $placeCards.addClass(options.selectors.noOriginClass);
+        var newPlaces = HomeTemplates.destinations(destinations);
+        $(options.selectors.placesContent).html(newPlaces);
+
+        // also draw on explore map
+        if (tabControl.isTabShowing(tabControl.TABS.EXPLORE) && mapControl.isLoaded()) {
+            mapControl.isochroneControl.drawDestinations(destinations);
+        }
+
+        showPlacesContent();
+
+        // now places list has been updated, go fetch the travel time
+        // from the new origin to each place
+        getTimesToPlaces();
+    }
```

Comment:
Is there anything happening here that's different from what `_getNearbyPlaces` is doing in its callback? (or different in an incompatible way--showing the spinner and hiding the existing cards is done before the lookup in `_getNearbyPlaces`, but doing it twice wouldn't hurt anything, and this is already the second call to `showSpinner`, since it's at the top of `_fetchIsochrone` as well)

Seems like this could be a `listPlaces(destinations)` function that could be used by both `_fetchIsochrone` and `_getNearbyPlaces`.  And given how much this little bit of code is doing and how many components it connects to, it would be nice to have it defined only once.